opentimelineio/opentime.py

```diff
@@ -620,7 +623,7 @@ def from_timecode(timecode_str, rate):
                 timecode_str, nominal_fps - 1))
 
     dropframes = 0
-    if rate_is_dropframe:
+    if rate_is_dropframe and treat_as_df:
```

Comment:
`rate_is_dropframe` is already factored into `treat_as_df` in line 599, I think this can just be:
```suggestion
    if treat_as_df:
```

Reply:
Agree :)